include/flashinfer/gemm/fp4_gemm_template_sm120.h

```diff
@@ -0,0 +1,276 @@
+         Arch, OperatorClass, ThreadBlockShape, ClusterShape, EpilogueTileType, ElementAccumulator,           \
+         ElementCompute, ElementC, LayoutC, AlignmentC, OutElementType, LayoutC, AlignmentC,                  \
+         EpilogueSchedule,                                                                                    \
+         cutlass::epilogue::fusion::LinearCombination<OutElementType, float, void,                            \
+                                                      float>>::CollectiveOp;                                  \
+                                                                                                              \
+       /* SM120 BlockScaled - Use nv_float4_t without tuples like example 79 */                                 \
+   /* Use fixed 2 stages for SM120 to meet minimum requirement */                                           \
+   using CollectiveMainloop = typename cutlass::gemm::collective::CollectiveBuilder<                        \
+       Arch, OperatorClass,                                                                                 \
+       ElementA, LayoutA, AlignmentA,                                                                       \
+       ElementB, LayoutB, AlignmentB,                                                                       \
+       ElementAccumulator,                                                                                  \
+       ThreadBlockShape, ClusterShape,                                                                      \
+       cutlass::gemm::collective::StageCount<2>,  /* Fixed 2 stages for SM120 */                           \
+       cutlass::gemm::collective::KernelScheduleAuto>::CollectiveOp;                                        \
+                                                                                                              \
+       /* Match example 79 - no wrapper needed */                                                               \
+   using GemmKernel = cutlass::gemm::kernel::GemmUniversal<                                                 \
+       cute::Shape<int, int, int, int>,  /* Indicates ProblemShape */                                       \
+       CollectiveMainloop,                                                                                  \
+       CollectiveEpilogue,                                                                                  \
+       void>;  /* No persistent scheduler for SM120 */                                                      \
+                                                                                                              \
+     using Gemm = typename cutlass::gemm::device::GemmUniversalAdapter<GemmKernel>;                           \
+   };                                                                                                         \
```

Comment:
[nit] // struct DeviceGemmFp4GemmSm120_##T##_##CTA_M_##_##CTA_N_##_##CTA_K_##_##CGA_M_##_##CGA_N_##_##CGA_K_##XSM_ 

to indicate end of struct definition